src/main/java/duke/exception/SavingError.java

```diff
@@ -0,0 +1,7 @@
+package duke.exception;
+
+public class SavingError extends DukeException {
```

Comment:
I like how you separate exception classes into subgroups to make the structure even cleaner